packages/core/src/storage/migration/__tests__/UpdateAssistant.test.ts

```diff
@@ -54,17 +54,25 @@ describe('UpdateAssistant', () => {
     it('should return true when a new wallet is created', async () => {
       expect(await updateAssistant.isUpToDate()).toBe(true)
     })
+
+    it('should return true for a lower version than current storage', async () => {
+      expect(await updateAssistant.isUpToDate('0.2')).toBe(true)
+    })
+
+    it('should return true for current agent storage version', async () => {
+      expect(await updateAssistant.isUpToDate('0.3')).toBe(true)
+    })
   })
 
   describe('UpdateAssistant.frameworkStorageVersion', () => {
     it('should return 0.2', async () => {
```

Comment:
```suggestion
    it('should return 0.3', async () => {
```
(or `CURRENT_FRAMEWORK_STORAGE_VERSION`, see my other comment)